src/components/ToggleSwitch/index.tsx

```diff
@@ -118,8 +118,8 @@ const ToggleSwitch: React.FC<ToggleSwitchProps> = ({
   labelRight,
   circleActiveColor = COLORS.anon_primary,
   circleInActiveColor = COLORS.signed_primary,
-  backgroundActive = COLORS.lightgrey,
-  backgroundInactive = COLORS.lightgrey,
+  backgroundActive = COLORS.almostBlack,
+  backgroundInactive = COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `backgroundActive` and `backgroundInactive` colors have been changed from `COLORS.lightgrey` to `COLORS.almostBlack`. As mentioned earlier, ensure that this color change aligns with the overall theme and accessibility standards of your application.
```diff
-  backgroundActive = COLORS.lightgrey,
-  backgroundInactive = COLORS.lightgrey,
+  backgroundActive = COLORS.almostBlack,
+  backgroundInactive = COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->